_posts/2015-02-18-ApiBlueprint-button.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Social blueprint
+excerpt: Share your blueprint with others and learn from masters.
+layout: post
+date: 2015-02-18 00:00:00 +0100
+author: vincenzo
+published: true
+commets: true
+---
+
+In the last times we have received a lot of feedback from users that would like to learn more about APIBlueprint and how to present stunning and beautiful documentation for their users.
+
+We care about your feedback and we think that documentation and tutorials about [APIBlueprint](http://apiblueprint.org) are extremely important. Given that we're hard working on this, and that tons of stuff is coming, today I am glad to announce that we've made another step forward in that direction.
+
+One of the most requested feature was the ability to look to the other APIBlueprint as a source of inspiration from that, and today this is possible, according to your APIBlueprint sharing policies.
```

Comment:
was to see the source of other API blueprints. 

Note: 
- API Blueprint (capitalized): name of the language
- API blueprint or just a blueprint: an API description – file – one particular blueprint, written using the API Blueprint language